src/components/pill/pill.stories.js

```diff
@@ -10,6 +10,16 @@ import classic from '../../style/themes/classic';
 import { THEMES } from '../../style/themes';
 import { notes, Info } from './documentation';
 
+const getStatusKnobs = () => {
+  const role = select('role', [...OptionsHelper.pillRoles], 'tag');
+
+  return {
+    role,
+    colourVariant: (role === 'status') ? select('colourVariant',
+      [...OptionsHelper.pillColours, OptionsHelper.colors[7]], OptionsHelper.pillColours[0]) : null
```

Comment:
Same as comment below re `colour` and `colourVariant` - `pillColours` should be Americanised for consistency

Reply:
`pillColours` occurrences still need updating